inspirehep/config.py

```diff
@@ -1437,7 +1438,7 @@ def _(x):
             },
             "source": {
                 "terms": {
-                    "field": "metadata.acquisition_source.method",
+                    "field": "metadata.acquisition_source.source",
```

Comment:
There's a bit of confusion on what the 'source' and the 'method' is, for forms for example, the method and the source are the same, but for hepcrawl they are not (that makes sense).

Maybe we should change the method of the submissions for something like 'forms' at some point.